src/Entity/RateTimeInTransit/EstimatedArrival.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Ups\Entity\RateTimeInTransit;
+
+use Ups\Entity\EstimatedArrivalTrait;
+
+class EstimatedArrival
+{
+    use EstimatedArrivalTrait;
+
+    protected $businessDaysInTransit;
+
+    /**
+     * @param \stdClass|null $response
+     */
+    public function __construct(\stdClass $response = null)
+    {
+        if (null !== $response) {
+            self::build($response);
```

Comment:
I'm not 100% sure if this is correct as the trait defines it as non-static. Didn't you mean `$this->build($response)`? Same for some other locations.